Gordon360/Documentation/Gordon360.xml

```diff
@@ -560,6 +554,15 @@
             <returns>The updated news item</returns>
             <remarks>The news item must be authored by the user and must not be expired and must be unapproved</remarks>
         </member>
+        <member name="M:Gordon360.Controllers.NewsController.EditPostingImage(System.Int32,System.String)">

```

Comment:
The changes to this file seem unrelated.  Do you know why they are included in this PR?